.github/workflows/qa.yml

```diff
@@ -0,0 +1,118 @@
+name: QA
+
+on:
+  workflow_dispatch:
```

Comment:
Should we run this on release as well?

Reply:
Or we can we do it manually before the release to catch any regressions?